Iryna_Bondarenko/greet.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+ module.exports = {
+	greetme: function (name) {
+		return "hello " + name;
+	}
+}
```

Comment:
Generally, we use two spaces for our tabs, it looks like you are using four.  This will cause major headaches when you start working with a group, so switch to 2 while we are still early in the course.  If you need help on how to do that in Sublime, come see me!